iwyu_include_picker.h

```diff
@@ -81,6 +81,8 @@ class IncludePicker {
   };
   // map_from to <map_to,...>
```

Comment:
Comments should be full sentences.

Reply:
Done (although this comment was there before I agree it's not a good comment; happy to tidy it up while I'm there).